cloud/profitbricks/profitbricks.py

```diff
@@ -561,12 +582,16 @@ def main():
             image=dict(),
             cores=dict(default=2),
             ram=dict(default=2048),
+            cpu_family=dict(default='AMD_OPTERON'),
```

Comment:
please add the choice list e.g.:

~~~python
cpu_family=dict(default='AMD_OPTERON', choices=[ 'AMD_OPTERON', 'INTEL_XEON']), 
~~~